.pre-commit-config.yaml

```diff
@@ -39,22 +39,22 @@ repos:
         types_or: [yaml, html, json]
 
   - repo: https://github.com/adamchainz/blacken-docs
-    rev: "1.16.0"
+    rev: "1.19.1"
     hooks:
       - id: blacken-docs
         additional_dependencies: [black==23.7.0]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: "v1.8.0"
+    rev: "v1.15.0"
     hooks:
       - id: mypy
         files: jupyter_core
         stages: [manual]
-        args: ["--install-types", "--non-interactive"]
-        additional_dependencies: ["traitlets>=5.13", "platformdirs>=3.11"]
```

Comment:
`"--install-types", "--non-interactive"` will actually silence some errors are they are ment to be run locally, I  think this is a mypy bug.